plugins/services/src/js/utils/DeclinedOffersUtil.js

```diff
@@ -35,6 +36,26 @@ const constraintsReducer = (accumulator, constraint) => {
 };
 
 const DeclinedOffersUtil = {
+  getAppsWithDeclinedOffersFromServiceTree(serviceTree = {}) {
+    if (!serviceTree.list || serviceTree.list.length === 0) {
```

Comment:
Please don't operate on internal "private" properties and use the public interface instead!